boards/imix/src/main.rs

```diff
@@ -702,6 +702,8 @@ pub unsafe fn main() {
     );*/
     //virtual_alarm_timer.set_alarm_client(mux_timer);
 
+    test::sha256_test::run_sha256(dynamic_deferred_caller);
```

Comment:
If we use the generic test framework here then the test won't be run on every boot (where users might not want to waste time running SHA tests)

Reply:
Where is the generic test framework documented, so I can use it? This is the way that all of the tests are currently done on imix. Or are you suggesting I incorporate a test on QEMU so it can be tested in the CI?

Reply:
The intention is this test is commented out for the final PR.

Reply:
Removed.